doc/examples/datasets_train/datasets_train.py

```diff
@@ -373,7 +373,10 @@ def train_func(config):
 
         # Checkpoint model.
         if is_distributed:
-            train.save_checkpoint(model_state_dict=net.module.state_dict())
+            import copy
+            model_copy = copy.deepcopy(net.module)
```

Comment:
Is the copy necessary?

Reply:
Yeah since `.cpu()` modifies the model. Alternatively I could map it back with `.to(device)` after but I wasn't sure if that's more efficient than copying.